tests/acceptance/load-all-test.js

```diff
@@ -11,38 +10,46 @@ function t(...args) {
 }
 
 async function domHasChanged(selector) {
-  let previousUi = find(selector).text();
+  let previousUi = find(selector).textContent;
   return await waitFor(() => {
-    let currentUi = find(selector).text();
+    let currentUi = find(selector).textContent;
 
     return currentUi !== previousUi;
   })
 }
 
-test('visiting /load-all', async function(assert) {
-  server.create('post', { id: '1', title: 'Post 1 title' });
-  server.create('post');
+module('Acceptance | load all', function(hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.afterEach(function() {
+    server.shutdown();
+  });
```

Comment:
Need to shutdown mirage after each test run. I know some folks are making their own moduleForAcceptance that includes this. Wasn't sure what was best here so I went with the 0 abstraction approach.

Reply:
yep I still need to figure this out in mirage. This looks good for now